src/auto-merge.js

```diff
@@ -19,6 +19,26 @@ module.exports = (robot) => {
     return handler(context)
   })
 
+  safeBind(['status'], context => {
+    const branches = context.payload.branches
+
+    if (branches.length !== 1) {
```

Comment:
Do you know when there would be multiple branches or PR's in the payload?

Reply:
if a commit exists on multiple branches. this API is not for PRs but for all commits, so that is possible, but in that case i probably don't care about the change